components/home/Home.bs

```diff
@@ -20,11 +21,80 @@ sub init()
 
     m.homeRows = m.top.findNode("homeRows")
     m.homeRows.focusBitmapBlendColor = chainLookupReturn(m.global.session, "user.settings.colorCursor", ColorPalette.HIGHLIGHT)
+
```

Comment:
Why create a new background element only for the home screen when we already have a background image that is used across the entire app?